sdk/src/types/block/error.rs

```diff
@@ -182,8 +185,8 @@ impl fmt::Display for Error {
             Self::InvalidOutputAmount(amount) => write!(f, "invalid output amount: {amount}"),
             Self::InvalidOutputCount(count) => write!(f, "invalid output count: {count}"),
             Self::InvalidOutputKind(k) => write!(f, "invalid output kind: {k}"),
-            Self::InvalidParentCount(count) => {
-                write!(f, "invalid parents count: {count}")
+            Self::InvalidParentCount => {
```

Comment:
Why remove the count?

Reply:
https://github.com/iotaledger/iota-sdk/pull/671/files/50728389683378681d33ff3777199b4de01fce0c#diff-334f5ed764795a38029cdd22cd2ec408970f26ec1379048c18b4e68fe186634dR62